sw/cheri/tests/pinmux_tests.hh

```diff
@@ -490,7 +330,7 @@ void pinmux_tests(CapRoot root, Log &log) {
 
   Capability<volatile SonataGpioFull> gpio_full = root.cast<volatile SonataGpioFull>();
   gpio_full.address()                           = GPIO_ADDRESS;
-  gpio_full.bounds()                            = GPIO_FULL_BOUNDS;
+  gpio_full.bounds()                            = GPIO_BOUNDS * GPIO_NUM;
```

Comment:
```suggestion
  gpio_full.bounds()                            = GPIO_RANGE * GPIO_NUM;
```

Reply:
See below comment; this change is now redundant due to the change to using a struct of capabilities instead.